homeassistant/package_constraints.txt

```diff
@@ -18,6 +18,7 @@ hass-nabucasa==0.54.0
 home-assistant-frontend==20220504.1
 httpx==0.22.0
 ifaddr==0.1.7
+isodate==0.6.1
```

Comment:
Meh, I don't like adding a new dependency. It's pulling in more code and only for a template function. The code you're linking is showing timezone parsing, but that's not part of a duration at all ?

Reply:
Thanks for the feedback. I will dive in and pull out the good stuff.